.github/workflows/bundle-size.yml

```
@@ -27,7 +27,7 @@ jobs:
       - run: yarn install
 
       - name: Build packages
-        run: yarn build
+        run: yarn build --filter=\!@strapi/design-system-docs
```

Just a small suggestion to all these `.github/workflows/*.yml` files:

```suggestion
        run: yarn build:ci
```

No strong motivation.
Not a blocking issue at all.

I just think that:
By having this new script `build:ci` in `package.json`, we will have all the arguments (--filter, in this case) into this single script, which can be easier to maintain for further potential changes.

WDYT?

---

I think it's an interesting idea for sure, if we were doing more than one filter i'd probably say yeah we should do it, but on the other hand abstracting it means someone has to look one more place to change it and with no testing ability you can't ensure there's no knock on effect... tricky balancing act imo.